.github/workflows/build.yaml

```diff
@@ -0,0 +1,18 @@
+---
+on: [push, pull_request]
+
+jobs:
+  compile:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - run: npm install
+      - run: npm run build
+      - run: npm run format
+      - uses: fregante/setup-git-user@v1
```

Comment:
```suggestion
```
I think this is useless, git is already setup

Reply:
No it's not: https://github.com/liyishuai/actions-comment-pull-request/runs/5620916317?check_suite_focus=true#step:7:6
```
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <runner@fv-az292-116.tqgtyn10aqxuxoi0mkfbldi5vh.jx.internal.cloudapp.net>) not allowed
Error: Process completed with exit code 12[8](https://github.com/liyishuai/actions-comment-pull-request/runs/5620916317?check_suite_focus=true#step:7:8).
```